workalendar/america/el_salvador.py

```diff
@@ -0,0 +1,25 @@
+from ..core import WesternCalendar
+from ..registry_tools import iso_register
+
+
+@iso_register('SV')
+class ElSalvador(WesternCalendar):
+    "El Salvador"
+    # Civil holidays
+    include_labour_day = True
+    # Christian holidays
+    include_holy_thursday = True
+    include_good_friday = True
+    include_easter_saturday = True
+
+    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
+        (5, 10, "Mothers' Day"),
+        (6, 17, "Fathers' Day"),
+        (8, 6, "Celebrations of San Salvador"),
+        (9, 15, "Independence Day"),
+        (11, 2, "All Saints Day"),
+    )
+
+    def get_variable_days(self, year):
```

Comment:
well, if it's only to use the "super()" method and return its result, you can remove the lines 21-25, then.